pkgs/tools/admin/copilot-cli/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "copilot-cli";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "copilot-cli";
+    rev = "v${version}";
+    sha256 = "1r84sim4fy58sdkpi47a8k7zx9ybxi84bdsa0kdb60ir4dig2ga6";
+  };
+
+  vendorSha256 = "1px4z747bnv1igxq6y4hvnfnn9y3r6hnb0q2sfk21mg8w0aj249c";
+
+  buildPhase = ''
+    make VERSION=${src.rev} compile-local
+  '';
```

This could be done using `makeFlags` as well.
```suggestion
  makeFlags = [ "VERSION=${src.rev}" "compile-local" ];
```

---

Thanks, I tried this out, but I'm getting a strange error when building with nixpkgs-review:
```
error: pkgs/tools/admin/copilot-cli/default.nix: No such file or directory
```
This might not be related, but it got me thinking... is `makeFlags` maybe only part of `mkDerivation`, and not `buildGoModule`?

---

Please use preBuild to keep 
```suggestion
buildFlagsArray = [ "-ldflags=-s -w -X github.com/aws/copilot-cli/internal/pkg/version.Version=${version} -X github.com/aws/copilot-cli/internal/pkg/cli.binaryS3BucketPath=https://ecs-cli-v2-release.s3.amazonaws.com" ];
```